sound/soc/sof/ipc4-topology.c

```diff
@@ -111,6 +111,12 @@ static const struct sof_topology_token gain_tokens[] = {
 		get_token_u32, offsetof(struct sof_ipc4_gain_data, init_val)},
 };
 
+/* SRC */
```

Comment:
typos in commit message: 
-needs two parameters (avoid abbreviations)
- alderlake

Don't mention PR, this is a patch.

Reply:
sure, thanks

Reply:
the commit message still says "PR"

Reply:
Thanks, fix it.